docs/mp/integrations/wls.adoc

```diff
@@ -0,0 +1,392 @@
+* The Oracle WebLogic Server and Helidon integration allows communication between a WebLogic cluster-hosted application and a Helidon microservice application by implementing Single Sign-on (SSO) authentication using Oracle Identity Cloud Service (IDCS).
+
+
+== REST Services Integration [[REST-Services]]
+
+- <<REST-Services-Overview, Overview>>
+- <<REST-Services-Maven-Coordinates, Maven Coordinates>>
+- <<REST-Services-Usage, Usage>>
+- <<REST-Services-References, References>>
+
+=== Overview [[REST-Services-Overview]]
+The REST services integration with Helidon enables bidirectional REST calls between Helidon and Oracle WebLogic Server. The Oracle WebLogic Server applications and Helidon microservices communicate through RESTful Web Service invocations, in one or both directions.
+
+=== Maven Coordinates [[REST-Services-Maven-Coordinates]]
+
+To enable Oracle WebLogic Server REST Integration, add the following dependencies to your project’s `pom.xml` file:
+
+[source,]
+----
+<dependency>
+    <groupId>io.helidon.microprofile.bundles</groupId>
+    <artifactId>helidon-microprofile-core</artifactId>
+</dependency>
+<dependency>
+    <groupId>com.fasterxml.jackson.core</groupId>
+    <artifactId>jackson-databind</artifactId>
+    <version>2.14.1</version>
+</dependency>
+<dependency>
+    <groupId>org.junit.jupiter</groupId>
+    <artifactId>junit-jupiter-api</artifactId>
+    <scope>test</scope>
+</dependency>
+----
+
+=== Usage [[REST-Services-Usage]]
+
+After you add the dependencies, you can build the Helidon client and invoke the Oracle WebLogic Web Service by compiling the Helidon client using Maven.
+[source,]
+----
+mvn clean package -DskipTests=true
+----
+For information about developing the REST Service using JAX-RS and Helidon API, see https://docs-uat.us.oracle.com/en/middleware/standalone/weblogic-server/14.1.1.0/wlshe/integrating-oracle-weblogic-server-rest-services-helidon.html#GUID-E9A7B731-ECCE-400A-AA18-FA07190C1AD2[Integrating WebLogic REST Services with Helidon 3.x].
+
+After you have compiled, start the Helidon server using the following command:
+[source,]
+----
+java -jar ./target/helidon-test.jar
+----
+Start the Helidon server to access the REST service running in Oracle WebLogic Server by using the Helidon URL. The URL will  use the name and port number of the host where the Helidon microservice application is running.
+[source,]
+----
+http://<HELIDON_HOST>:<HELIDON_PORT>/client/<handler>
+----
+To test the application, send a request from the Helidon endpoint using one of the following:
+
+* A Java client
+* A browser
+* Postman 
+
+You should receive a response from the REST service deployed in Oracle WebLogic Server. For example, you should receive the following response when you initiate the `getHelidonClientToWlsRestText` handler:
+[source,]
+----
+Hi there!
+----
```

Comment:
It looks weird to me. No sample is explained, but there is information about how to start it and what it should return.